src/TsGenerator.php

```diff
@@ -97,7 +97,7 @@ protected function generateApiOperationMap(array $data, array $modules): void
                 }
```

Comment:
one more issue: when generate operations and module is 2 words (like offering_cms) - the file name for operations is operations_cmsOperations, should be all in camel case